sdk/web-pubsub/web-pubsub/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/web-pubsub",
-  "version": "1.0.0-beta.4",
+  "version": "1.0.0",
   "description": "Azure client library for Azure Web PubSub",
```

Comment:
also update homepage at L71 `"homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/search/search/",`

Reply:
`"@azure/core-tracing"` update to stable version?

Reply:
There isn't a stable core-tracing yet. I've updated the homepage link, good catch!